bundles/alpha.glue.v2/export.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project default="create_run_jar" name="Create Runnable Jar for Project alpha.glue.v2 with Jar-in-Jar Loader">
+    <!--this file was created by Eclipse Runnable JAR file Export Wizard-->
+    <!--ANT 1.7 is required-->
+    <!--define folder properties-->
+    <property name="dir.buildfile" value="."/>
+    <property name="dir.jarfile" value="${basedir}/../../artifact/bin"/>
+    <target name="create_run_jar">
+        <jar destfile="${dir.jarfile}/alpha.glue.v2.jar">
+            <manifest>
+                <attribute name="Main-Class" value="org.eclipse.jdt.internal.jarinjarloader.JarRsrcLoader"/>
+                <attribute name="Rsrc-Main-Class" value="alpha.glue.v2.GenerateNewWriteC"/>
+                <attribute name="Class-Path" value="."/>
+            	<attribute name="Rsrc-Class-Path" value="./ org.eclipse.emf.ecore.jar org.eclipse.emf.common.jar org.eclipse.xtext.xbase.lib.jar com.google.guava.jar org.eclipse.emf.ecore.xcore.lib.jar org.eclipse.core.runtime.jar org.eclipse.osgi.jar org.codehaus.groovy.eclipse.frameworkAdapter.jar org.eclipse.osgi.compatibility.state.jar org.eclipse.equinox.common.jar org.eclipse.core.jobs.jar org.eclipse.equinox.registry.jar org.eclipse.equinox.preferences.jar org.osgi.service.prefs.jar org.eclipse.core.contenttype.jar org.eclipse.equinox.app.jar org.eclipse.emf.ecore.xcore.jar org.eclipse.xtext.jar org.eclipse.emf.ecore.xmi.jar org.antlr.runtime.jar com.google.inject.jar org.eclipse.emf.mwe.core.jar org.apache.commons.cli.jar org.eclipse.emf.mwe2.runtime.jar org.eclipse.emf.mwe.utils.jar org.eclipse.xtext.util.jar javax.inject.jar org.apache.log4j.jar org.eclipse.xtext.xbase.jar org.eclipse.xtext.common.types.jar org.eclipse.xtend.lib.jar org.eclipse.xtend.lib.macro.jar org.eclipse.xtext.ecore.jar org.eclipse.emf.codegen.ecore.xtext.jar fr.irisa.cairn.jnimap.isl.jar fr.irisa.cairn.jnimap.runtime.jar org.eclipse.xtend.typesystem.emf.jar fr.irisa.cairn.jnimap.polylib.jar fr.irisa.cairn.jnimap.barvinok.jar alpha.model.xtext.jar alpha.targetmapping.xtext.jar alpha.loader.jar alpha.targetmapping.jar"/>
+            </manifest>
+            <zipfileset src="jar-in-jar-loader.zip"/>
+            <fileset dir="${dir.buildfile}/bin"/>
+            <fileset dir="${basedir}../../../alpha-language/bundles/alpha.codegen/bin"/>
+            <fileset dir="${basedir}../../../alpha-language/bundles/alpha.model/bin"/>
+            <mappedresources>
+                <zipfileset dir="${ant.home}/../">
+                	<include name="org.eclipse.emf.ecore_2.27.0.v20220426-0617.jar"/>
```

Comment:
All of the v2 exports still have the build data/time. I’m guessing you were waiting to see if the v1 updates works.

Reply:
Ah, no I just missed that somehow. Fixed now